src/Controls/tests/UITests/Tests/Issues/_IssuesUITest.cs

```diff
@@ -9,15 +10,41 @@ public abstract class _IssuesUITest : UITestBase
 
 		protected override void FixtureSetup()
 		{
-			base.FixtureSetup();
-			NavigateToIssue(Issue);
+			int retries = 0;
+			while (true)
+			{
+				try
+				{
+					base.FixtureSetup();
+					NavigateToIssue(Issue);
+					break;
+				}
+				catch (Exception)
+				{
+					if (retries++ < 1)
```

Comment:
So it retries one time? Maybe change the 1 to a MaxRetries const (the code looked odd to me at first glance).